core/src/main/java/com/novoda/noplayer/mediaplayer/AndroidMediaPlayerImpl.java

```diff
@@ -261,33 +260,28 @@ public void detach(PlayerView playerView) {
     }
 
     @Override
-    public boolean selectAudioTrack(PlayerAudioTrack audioTrack) {
+    public boolean selectAudioTrack(PlayerAudioTrack audioTrack) throws IllegalStateException {
         return mediaPlayer.selectAudioTrack(audioTrack);
     }
 
     @Override
     public boolean showSubtitleTrack(PlayerSubtitleTrack subtitleTrack) {
-        SubtitlesError subtitlesError = new SubtitlesError("Subtitles not implemented for Android Media Player", new IllegalStateException());
-        listenersHolder.getErrorListeners().onError(this, subtitlesError);
```

Comment:
Yes. After talking to Adam, it's not worth reporting in this way. It's not supported in `MediaPlayer`but it should be in the future. I would like to have a wider array of `Capabilities` that `Player`s can support and then we select a `Player` based on that.